Formula/got.rb

```diff
@@ -0,0 +1,22 @@
+class Got < Formula
+  desc "Version control system"
+  homepage "https://gameoftrees.org"
+  url "https://gameoftrees.org/releases/portable/got-portable-0.75.tar.gz"
+  sha256 "b35324893c9aefbd6bf49602a6c9bb9d6e20670aff3c9734fefead0205bb9cb2"
+  license "ISC"
+
+  depends_on "ncurses" => :build
+  depends_on "openssl@3" => :build
+  depends_on "ossp-uuid" => :build
```

Comment:
These are likely needed at runtime. Does macOS `ncurses` not work?

Reply:
> These are likely needed at runtime.

Yep, brainfart on my part.

> Does macOS ncurses not work?

`uses_from_macos "ncurses"` doesn't work. The configure script wants `ncursesw` and `panelw`, which I think are not provided by macOS ncurses.